odata-csdl-json/meta.yaml

```diff
@@ -0,0 +1,6 @@
+pagetitle: OData Common Schema Definition Language (CSDL) JSON Representation Version 4.02
+subtitle: Committee Specification Draft 01
+filename: odata-csdl-json-v4.02-csd01
```

Comment:
filename deviates from pattern used in other meta.yaml, eg for odata-protocol

Reply:
See above